bdsx/event_impl/entityevent.ts

```diff
@@ -548,3 +561,13 @@ const sendActorCarriedItemChanged = procHacker.hooking(
     decay(newItemStack);
     return sendActorCarriedItemChanged(self, entity, oldItemStack, newItemStack, handSlot);
 });
+
+function onEntityKnockback(target: Actor, source: Actor | null, damage: int32_t, xd: float32_t, zd: float32_t, power: float32_t, height: float32_t, heightCap: float32_t): void {
```

Comment:
what do you think about set `target` as a `Mob`?

Reply:
Yep ! also fixed it for `EntityDieEvent` which hooks `Mob::die`

Reply:
nice work